daemon/src/bin/fuzzer.rs

```diff
@@ -33,6 +33,8 @@ fn initialize_project() -> (temp_dir::TempDir, PathBuf) {
     (dir, file)
 }
 
+/* TODO: consider moving fuzzer to separate crate to avoid adding test code to ethersync binary. */
```

Comment:
Maybe an actual [integration test](https://doc.rust-lang.org/rust-by-example/testing/integration_testing.html) would be the correct place for this! This would also avoid the problem of having a "fuzzer" binary in our output if you run `cargo build`. I created #128 to track this. Feel free to remove this TODO :)

Reply:
I ended up spending some time doing almost exactly this, but had created a subcrate instead of an integration test (https://github.com/ethersync/ethersync/compare/main...cosmicexplorer:use-cargo-test-for-integration-tests?expand=1). I was literally about to post a separate PR for it when I saw your review of this one -- I'll update that PR to make it an integration test instead!

Reply:
Done! https://github.com/ethersync/ethersync/pull/129